LayoutTests/fast/forms/meter-appearance-on-pseudo-element.html

```diff
@@ -0,0 +1,18 @@
+<style>
+    dialog::backdrop {
+        content: "";
```

Comment:
I realize content: "" is not needed on ::backdrop (I originally wrote this for ::marker)

Reply:
Removed.